ci/bmiconfig.cfg.ci

```diff
@@ -0,0 +1,70 @@
+# this section is for basic bmi settings
+[bmi]
+# uid is given so that images dont clash in ceph pool
+uid = 1
+service = False
+
+# this section is for db settings
+[db]
+path = /var/lib/bmi/bmi.db
+
+# This section is filesystem related config
+[fs]
+id = admin
+pool = rbd
+conf_file = /etc/ceph/ceph.conf
+keyring = /etc/ceph/ceph.client.admin.keyring
+
+[driver]
+net_isolator = <driver to load>
+iscsi = <driver to load>
+fs = <driver to load>
+
+# This section is for network isolator (HIL) related config
+[net_isolator]
+url = http://172.17.0.3:7000
```

Comment:
Do we know if this IP is guaranteed? I think we can assign this as a static ip when we run the container

Reply:
In our CI we start the the ceph container first which gets the ip of 172.17.0.2 (see MON_IP in .travis.yml when launching the ceph container). And then we launch the hil container which gets the next ip which is 172.17.0.3. Right now these ips are not statically assinged (since if you launch the hil container first it gets 172.17.0.2). We can change to static assignment in a different PR.

Reply:
ah ok, I was wondering if the ips were sequential like that.  We can revisit the static bit later